app/models/pipeline_run_stage.rb

```diff
@@ -25,10 +23,19 @@ def install_pipeline
     "pip install -e .[test]"
   end
 
-  def aegea_batch_submit_command(base_command, memory = Sample::DEFAULT_MEMORY)
+  def aegea_batch_submit_command(base_command, memory = Sample::DEFAULT_MEMORY_IN_MB)
     command = "aegea batch submit --command=\"#{base_command}\" "
-    queue = pipeline_run.sample.job_queue.present? ? pipeline_run.sample.job_queue : Sample::DEFAULT_QUEUE
-    command += " --storage /mnt=#{DEFAULT_STORAGE_IN_GB} --ecr-image idseq --memory #{memory} --queue #{queue} --vcpus 4"
+    if memory <= Sample::DEFAULT_MEMORY_IN_MB
+      vcpus = Sample::DEFAULT_VCPUS
+      queue = Sample::DEFAULT_QUEUE
+    else
+      vcpus = Sample::DEFAULT_VCPUS_HIMEM
+      queue = Sample::DEFAULT_QUEUE_HIMEM
+    end
+    if pipeline_run.sample.job_queue.present?
+      queue = pipeline_run.sample.job_queue
+    end
+    command += " --storage /mnt=#{Sample::DEFAULT_STORAGE_IN_GB} --ecr-image idseq --memory #{memory} --queue #{queue} --vcpus #{vcpus}"
```

Comment:
I think the latest changes cleaned this up a bit better.   Does it still look good, @cdebourcy?

Reply:
It looks good to me!